src/gsi/cplr_wrwrfmassa.f90

```diff
@@ -2085,25 +2119,25 @@ subroutine wrwrfmassa_netcdf_wrf(this,mype)
        end if
        if ( laeroana_gocart ) then
           do iv = 1, n_gocart_var
-             i_chem(iv)=i_tt+(iv-1)*lm+1
+             i_chem(iv)=i_tt+(iv-1)*lm+1        ! ???? (tt is 3D field, should i_chem be i_tt + iv*lm ?)
```

Comment:
do you still need the comment here?

Reply:
@ShunLiu-NOAA 
Hi Shun,

This comment is not related to my modified code, and not necessary here. I am going to remove it  in the updated new code.

Thank you for pointing it out for me!